stable/gitlab-ce/requirements.lock

```diff
@@ -0,0 +1,9 @@
+dependencies:
+- name: redis
+  repository: http://storage.googleapis.com/kubernetes-charts
```

Comment:
The default repo changed to http://kubernetes-charts.storage.googleapis.com which is why CI is failing. 

Checkout:
#234